wis2node/resources/data-mappings.yml

```diff
@@ -1,3 +1,6 @@
 
 data:
     observations-surface-land.mw.FWCL.landFixed: wis2node.data.observations.ObservationData
+
+api_backend:
```

Comment:
As discussed, we should update the plugin and handling logic so that only the data mappings are required.  For a future PR.